Paco-Server/src/com/google/sampling/experiential/server/migration/MigrationFrontendServlet.java

```diff
@@ -62,43 +62,43 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
     DateTime stDate = null;
     DateTime endDate = null;
     cursor =  req.getParameter("cursor");
-    if (user == null) {
-      AuthUtil.redirectUserToLogin(req, resp);
-      
-    } else  if (AuthUtil.isUserAdmin()) { 
-      String jobName = req.getParameter("name");
-      String startTime = req.getParameter("startTime");
-      String endTime = req.getParameter("endTime");
-      try{
-        DateTimeFormatter formatter  = DateTimeFormat.forPattern(TimeUtil.DATE_TIME_WITH_NO_TZ);
-        if(startTime!=null && endTime != null) {
-          stDate = formatter.parseDateTime(startTime);
-          endDate = formatter.parseDateTime(endTime);
-        }
-      } catch (IllegalArgumentException e ){
-        resp.sendError(HttpServletResponse.SC_BAD_REQUEST);
+    String jobName = req.getParameter("name");
+    String startTime = req.getParameter("startTime");
```

Comment:
Since this servlet is in a frontend module and does admin work, how do we ensure that only admins can access it? Is the security constraint in web.xml sufficient?